WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.h

```diff
@@ -49,6 +49,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (nullable XCUIElement *)fb_activeElement;
 
+#if TARGET_OS_TV
```

Comment:
📝 
Will remove `#if TARGET_OS_TV` since normal iPhone also has it https://developer.apple.com/documentation/xctest/xcuielementattributes/1627636-hasfocus